apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/mottaker/BrevMottakerModal.tsx

```diff
@@ -15,18 +15,18 @@ enum MottakerType {
 }
 
 interface Props {
-  brev: IBrev
-  setBrev: Dispatch<SetStateAction<IBrev>>
+  brevId: number
+  sakId: number
+  mottaker: Mottaker
+  setMottaker: Dispatch<SetStateAction<Mottaker>>
 }
 
-export function BrevMottakerModal({ brev, setBrev }: Props) {
-  const { id: brevId, sakId, mottaker: initialMottaker } = brev
-
+export function BrevMottakerModal({ brevId, sakId, mottaker: initialMottaker, setMottaker }: Props) {
   const [mottakerStatus, apiOppdaterMottaker] = useApiCall(oppdaterMottaker)
 
   const [isOpen, setIsOpen] = useState(false)
   const [mottakerType, setMottakerType] = useState(
-    brev.mottaker.orgnummer ? MottakerType.BEDRIFT : MottakerType.PRIVATPERSON
+    initialMottaker.orgnummer ? MottakerType.BEDRIFT : MottakerType.PRIVATPERSON
```

Comment:
Legger merke til at det opereres med to ulike type `MottakerType`. En for frontend og en for backend, men det blir kanskje gjort noe med i senere PR?

Reply:
`MottakerType` i frontend er bare noe som brukes ved redigering av mottaker og persisteres aldri. Tenker den kan få et annet navn når den andre varianten blir innført. Må bare pønske på noe lurt... 😅